### ISSUE TITLE
Remove fork pull requests limitation

### ISSUE THREAD

**Issue by RomainBelorgey** [reviewdog/reviewdog#759]
https://github.com/reviewdog/reviewdog/blob/master/cmd/reviewdog/main.go#L260-L265

It is currently not possible to use `reviewdog` on a pull request from a private fork repository.

But looking at this action: https://github.com/tibdex/github-app-token, it is possible to create an app with the good permissions and reviewdog will have no issues.

**Comment by haya14busa** [comment]
Accessing secret itself is not allowed for fork pull request.

**Comment by RomainBelorgey** [comment]
> Accessing secret itself is not allowed for fork pull request.

Yes you can, you just need to specify it on the settings: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/

**Comment by haya14busa** [comment]
Do you know how to check the token have write permission or not?

**Comment by RomainBelorgey** [comment]
> Do you know how to check the token have write permission or not?

Sorry no idea, but it can be an option when running reviewdog and people need to check that they got the permissions ?
https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/ or an github app token with the good permissions can help them

**Comment by RomainBelorgey** [comment]
Is there maybe a way to ignore this https://github.com/reviewdog/reviewdog/blob/master/cmd/reviewdog/main.go#L283 and test if working fine ? Or should I find a way to download a custom build of reviewdog from the action ? :thinking:
